deploy/ui/form.json

```diff
@@ -1328,6 +1328,50 @@
               ]
             }
           ]
+        },
+        {
+          "label": "Process Instance Migration",
+          "fields": [
+            {
+              "label": "Image",
+              "required": false,
+              "jsonPath": "$.spec.objects.pim.image",
+              "description": "The image to use for Process Instance Migration.",
+              "type": "text"
+            },
+            {
+              "label": "Image tag",
+              "required": false,
+              "jsonPath": "$.spec.objects.pim.imageTag",
+              "description": "The image tag to use for Process Instance Migration.",
+              "type": "text"
+            },
+            {
+              "label": "Database type",
```

Comment:
The database size is missing

Reply:
Once changes to console UI is finished, I'll update this pull request to include these changes.

Reply:
Fields are still missing here